cli/e2e.js

```diff
@@ -150,10 +151,19 @@ function spawnSelenium() {
  */
 function spawnBuild(argv, skyPagesConfig, webpack) {
   return new Promise((resolve, reject) => {
+
+    if (argv.build === false) {
+      logger.info('Skipping build step');
+      return resolve({
+        metadata: fs.readJsonSync('dist/metadata.json')
```

Comment:
I know `fs` likes to throw errors around. Are we fairly confident that `metadata.json` will always exist? ie, we don't need to wrap this in a `try/catch`?